### ISSUE TITLE
loss issue

### ISSUE THREAD

**Issue by wsliveup** [lyp-deeplearning/LiftFeat#7]
This is a well-executed study, but I have some reservations regarding the implementation details. While the paper mentions only three loss functions, the code introduces an additional loss term through:

fb_coordinate = self.net.fine_matcher(torch.cat([fb_feat1, fb_feat2], dim=-1))

My primary concern is about the underlying assumption of pixel-wise correspondence in image pairs. Since real-world image pairs don't necessarily have perfect pixel-to-pixel matches, what is the actual purpose and theoretical justification for this additional loss term?

**Comment by HITer-lwp** [comment]
Thank you for your interest in our work. The coordinate loss is actually an additional component introduced in XFeat[https://github.com/verlab/accelerated_features]. We experimented with it during training and found that it had only a minor impact on the final results. Therefore, we did not include it in our final submission. However, we have now added a flag to enable or disable it, and the code has been updated accordingly.

**Comment by wsliveup** [comment]
Understood, thank you very much for your reply, I will go and read XFEAT again to deepen my understanding.
